### PR TITLE
How Machine Learning is Revolutionizing the Agricultural Industry 

### LINKED ISSUE #4374: [Machine Learning] Machine Learning in Agricultural Revolution
#### Proposed article introduction
*In the era of increased need for better decision making in agricultural production, there is a need for machine learning which is vital in analytics, security and management
Machine learning is a crucial branch of AI that is vital in the estimation and prediction of agricultural aspects such as quality of crops and yield
The article focuses on assessing the role of machine learning in agricultural production.

#### Key takeaways 
1.	Why is machine learning meaningful in agriculture? 
2.	How does machine learning able to make estimations and predictions of farm parameters?
3.	For example, how is machine learning more efficient compared to other technologies, especially in crop production?

#### Article quality
Most of the articles have focused on the prediction of yield production. However, this article goes further to assess the role of machine learning in the production process to comprise detection of pests, weeds, disease, and the quality of crops.  

#### References
Not applicable

### PR DESCRIPTION
**Pull request by Ann-tech348**
this closes #4374

### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [ ] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [ ] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #issue-number`

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by Ann-tech348** [comment]
Okay, i will make the changes

**Comment by Ann-tech348** [comment]
@srishilesh, I did the revision, kindly review

**Comment by srishilesh** [comment]
@Ann-tech348 

Thanks for making the changes.

Let's wait for this article to get plagiarism checked.

Thanks.

**Comment by srishilesh** [comment]
@section-engineering-education/content-moderator-team 

Can we have this PR plagiarism checked?

**Comment by Ann-tech348** [comment]
Is this in reference to the author?

**Comment by Ann-tech348** [comment]
"Can we have this PR plagiarism checked?"
Is this in reference to the author please?

**Comment by srishilesh** [comment]
> "Can we have this PR plagiarism checked?" Is this in reference to the author please?

No. It isn't.

The content moderators will be running a plagiarism check to see if the score is less than 10%.

**Comment by srishilesh** [comment]
> @section-engineering-education/content-moderator-team
> 
> Can we have this PR plagiarism checked?

@hectorkambow 

Can we have this PR plagiarism checked?

I have pinged the CMs as well, couldn't find any response.

**Comment by srishilesh** [comment]
@lalith1403 @Bayler @collins-a 

Need your help here.

If you find time, can you guys have a quick look at this article? Do you think any other topics must be added? Or how this article could be improved?

I'm a little doubtful content-wise, since these information aren't backed by enough data, supporting it. Most of the points addresses the question of "What?", but not "How?" or "Why?".

Your thoughts would definitely help me in deciding whether to ask the author to rework on this.

cc. @hectorkambow

**Comment by collins-a** [comment]
> @lalith1403 @Bayler @collins-a
> 
> Need your help here.
> 
> If you find time, can you guys have a quick look at this article? Do you think any other topics must be added? Or how this article could be improved?
> 
> I'm a little doubtful content-wise, since these information aren't backed by enough data, supporting it. Most of the points addresses the question of "What?", but not "How?" or "Why?".
> 
> Your thoughts would definitely help me in deciding whether to ask the author to rework on this.
> 
> cc. @hectorkambow

Hey @srishilesh, I do agree with you, it's a little light on the _how_. Let me go ahead and highlight a couple of areas.

File: content/articles/how-machine-learning-is-revolutionizing-the-agricultural-industry/index.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+#### Crop yield optimization
+
+Machine learning is vital in predicting crop yield through visual analytic data and real-time sensor data. The drones and smart sensors capture data that experts in the agricultural industry use in the analysis. The drawn and smart sensor captures data on fertilizer, moisture, and natural nutrient levels. The data is vital in assessing the crop's growth pattern and optimizing crop yields.
+
+Machine learning can combine different sets of data and advice on ways to optimize agricultural production. Machine learning can combine and regression and CART to improve the health and quality of products. The
```
Comment (collins-a) [inline content/articles/how-machine-learning-is-revolutionizing-the-agricultural-industry/index.md:52]: A reader would enjoy more value from these two points if they were a bit more technical. e.g. how does this ML surveillance system work? What makes it an enabler of a revolution? How does ML combine algos to improve agricultural output? Which algos, in particular, may be used in a given use case?

Alternatively, you could have a few more points that truly bring out ML enhancing the revolution.

**Comment by WanjaMIKE** [comment]
Plagiarism check done.

**Comment by srishilesh** [comment]
@collins-a Thanks for your suggestion. It was really helpful.

Let's give author sometime to make these changes.

---
cc. @Ann-tech348 We'll be closing this PR. Feel free to open a new PR, and mention me when the changes are done.

As mentioned in the suggestions, it would be great if you focus on the "how" part more. Explain some algo's in detail.

Feel free to ping me in case of any questions. Thanks.

**Comment by srishilesh** [comment]
@Ann-tech348 

Do we have any updates on this?

Just wanted to check if you need any help.

**Comment by Ann-tech348** [comment]
I am till working on it and will respond within the week. thanks

**Comment by Ann-tech348** [comment]
@srishilesh , i am unable to update the document since cant locate the folder on the visualstudio code, kindly help

**Comment by srishilesh** [comment]
> @srishilesh , i am unable to update the document since cant locate the folder on the visualstudio code, kindly help

@Ann-tech348 

I'm sorry, which document?

Probably, let's connect via slack.

**Comment by Ann-tech348** [comment]
I meant @srishilesh you open this PR so that I can finish this revisions

**Comment by srishilesh** [comment]
> @collins-a Thanks for your suggestion. It was really helpful.
> 
> Let's give author sometime to make these changes.
> 
> ---
> cc. @Ann-tech348 We'll be closing this PR. Feel free to open a new PR, and mention me when the changes are done.
> 
> As mentioned in the suggestions, it would be great if you focus on the "how" part more. Explain some algo's in detail.
> 
> Feel free to ping me in case of any questions. Thanks.

@Ann-tech348 

As I had mentioned previously, you would have to open a new PR. Until then you can make those revisions in your forked repository.

If we reopen this PR for you to make changes, it would be unfair to other contributors.

Ensure that your PR must be ready to be published.

**Pull request closed without merging**